src/main/java/org/testcontainers/jooq/codegen/database/DatabaseProvider.java

```diff
@@ -15,7 +16,8 @@ public static JdbcDatabaseContainer<?> getDatabaseContainer(DatabaseProps props)
         String image = Optional.ofNullable(props.getContainerImage()).orElse(dbType.getDefaultImage());
         JdbcDatabaseContainer<?> container =
                 switch (dbType) {
-                    case POSTGRES -> new PostgreSQLContainer<>(image);
+                    case POSTGRES -> new PostgreSQLContainer<>(
+                            DockerImageName.parse(image).asCompatibleSubstituteFor("postgres"));
                     case MARIADB -> new MariaDBContainer<>(image);
                     case MYSQL -> new MySQLContainer<>(image);
```

Comment:
Maybe add for others as well?
It might be helpful, somebody will want use custom images

Reply:
Done